include/selene/State.h

```diff
@@ -18,6 +18,39 @@ class State {
     std::unique_ptr<Registry> _registry;
     std::unique_ptr<ExceptionHandler> _exception_handler;
 
+    /// Shared code post-calling LoadString or LoadFile
+    inline bool PostLoad(int status, const char* file = nullptr) {
```

Comment:
the second parameter name should be changed to reflect that it may not necessarily be a file any more.